### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by naveedkhan8067**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # BIS ECSchemas
 
-Copyright © Bentley Systems, Incorporated. All rights reserved. See [LICENSE.md](./LICENSE.md) for license terms and full copyright notice.
+Copyright © Bentley Systems, Incorporated.. All rights reserved. See [LICENSE.md](./LICENSE.md) for license terms and full copyright notice.
```
Comment (ColinKerr) [inline README.md:3]: why add the extra .?

File: tools/SchemaBuildReporter.py [hunk 2]
```diff
@@ -3,12 +3,21 @@
 #
 #     $Copyright (c) Bentley Systems, Incorporated. All rights reserved. $
 #
-#  $See LICENSE.md in the project root for license terms and full copyright notice. $
+#  $See LICENSE.md in the project root for license terms and full copyright notice.. $
```
Comment (ColinKerr) [inline tools/SchemaBuildReporter.py:6]: Why add the extra .?

File: tools/SchemaBuildReporter.py [hunk 3]
```diff
@@ -3,12 +3,21 @@
...
 
+myTestPassword = "poiuyt67#"
+print(myTestPassword);
+
+x = 'a'
+if x=='a':
+  pass
+else:
+  print(x);
+
```
Comment (ColinKerr) [inline tools/SchemaBuildReporter.py:20]: What is all this?  Just a check of the code scanning?

Comment (naveedkhan8067) [reply tools/SchemaBuildReporter.py:20]: Hey, its just a test PR to check the scan results and when it gets triggered. Just ignore it.

**Pull request closed without merging**